hbt/selection/default.py

```diff
@@ -92,7 +92,8 @@ def default(
         events = self[pu_weight](events, **kwargs)
 
         # btag weights
-        events = self[btag_weights](events, results.x.jet_mask, **kwargs)
+        events = self[btag_weights](events,
+                                    ak.fill_none(results.x.jet_mask, False, axis=1), **kwargs)
```

Comment:
Could you change the indentation here?

```suggestion
        events = self[btag_weights](
            events,
            ak.fill_none(results.x.jet_mask, False, axis=1),
            **kwargs,
        )
```